src/PIL/JpegImagePlugin.py

```diff
@@ -463,6 +463,7 @@ def _getexif(self):
     head = file.read(8)
     # process dictionary
     info = TiffImagePlugin.ImageFileDirectory_v1(head)
+    file.seek(info.next)
```

Comment:
Unrelated, but whilst this method is being edited, shall we rename the `file` variable so it's not shadowing the builtin?

Reply:
Okay, I've pushed a commit.